src/pages/tutorial/task/task.js

```diff
@@ -246,12 +254,58 @@ class TaskPage extends React.Component {
                   <div className="integr8ly-module-column--footer">
                     <h6>{t('task.CompleteAndCheck')}</h6>
                     <div className="integr8ly-module-column--footer_status">
-                      <Icon type="fa" name="circle-o" />
-                      <span className="integr8ly-module-column--footer_status-step">1.1</span>
-                      <Icon type="fa" name="circle-o" />
-                      <span className="integr8ly-module-column--footer_status-step">1.2</span>
-                      <Icon type="fa" name="circle-o" />
-                      <span className="integr8ly-module-column--footer_status-step">1.3</span>
+                      {threadTask.steps.map((step, l) => (
+                        <React.Fragment key={l}>
+                          {step.infoVerifications &&
+                            step.infoVerifications.map(() => (
+                              <Icon
+                                className={
+                                  step.infoVerifications && verifications[step.infoVerifications[0]]
```

Comment:
the arrays for infoverifications and successverifications should be changed to scalars.